test/integration/conftest.py

```diff
@@ -121,31 +121,9 @@ def sagemaker_session(boto_session, sagemaker_client) -> Session:
 
 @pytest.fixture(scope="session")
 def is_feature_store_available(region) -> bool:
-    """Check if feature store is available in current region."""
+    """Check if feature store is available in current region. Setting list empty since FS library not available for
+    spark releases later than 3.3. Otherwise it will fail tests during release."""
```

Comment:
worth to send offline message to feature store team on this to make sure they are aware of this.

Reply:
Yes, I got this confirmation from @can-sun  and he requested me to disable test and installation of feature store library in the 3.4 release.